contracts/PriceOracle/PriceOracle.sol

```diff
@@ -35,21 +34,23 @@ contract PriceOracle is Ownable2Step {
 
     /// @notice The event emitted when an asset is removed to the mapping
     /// @param cToken cToken address that the config was removed for
-    /// @param baseUnit Number of smallest units of measurement in a single whole unit set in the removed config.
+    /// @param underlyingAssetDecimals Decimals of the underlying asset in the removed config.
     /// @param priceFeed Address price feed set in the removed config
-    event PriceOracleAssetRemoved(address indexed cToken, uint256 baseUnit, address priceFeed);
+    event PriceOracleAssetRemoved(address indexed cToken, uint256 underlyingAssetDecimals, address priceFeed);
 
     /// @notice The max decimals value allowed for price feed
     uint8 internal constant MAX_DECIMALS = 72;
 
     /// @notice The number of digits the price is scaled to before adjusted by the base units
     uint8 internal constant PRICE_SCALE = 36;
 
+    uint24 internal constant STALE_PRICE_DELAY_SECONDS = 87_000; // Staleness delay set to 24hrs + 10min buffer
```

Comment:
Can we make this configurable instead?

Reply:
Sure, I can do that. I can potentially make this configurable at a config level since it would vary feed to feed based on the heartbeat. Since this contract and adding new configs will be owned by Compound, I was worried about misconfigurations if they don't have insight into our feed's heartbeat config. I'll check with them on this though and at the least make the global staleness window configurable in case it needs to be adjusted.